ios/FluentUI/Avatar/AvatarTokenSet.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// Design token set for the `Avatar` control.
+open class AvatarTokenSet: ControlTokenSet<AvatarTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// The size of the content of the `Avatar`.
+        case avatarSize
+
+        /// The radius of the corners of the `Avatar`.
+        case borderRadius
+
+        /// The font used for text in the `Avatar`
+        case textFont
+
+        /// The default color of the ring around the `Avatar`.
+        case ringDefaultColor
+
+        /// The color of the gap between the ring and `Avatar`.
+        case ringGapColor
+
+        /// The thickness of the ring around the `Avatar`.
+        case ringThickness
+
+        /// The gap between the `Avatar` and its ring.
+        case ringInnerGap
+
+        /// The gap around the ring around the `Avatar`.
+        case ringOuterGap
+
+        /// The size of the presence icon.
+        case presenceIconSize
+
+        /// The thickness of the outline around the presence icon.
+        case presenceIconOutlineThickness
+
+        /// The color of the outline around the presence.
+        case presenceOutlineColor
+
+        /// The default color of the background of the `Avatar`.
+        case backgroundDefaultColor
+
+        /// The default color of the foreground of the `Avatar`
+        case foregroundDefaultColor
+
+    }
+
+    override func defaultValue(_ token: Tokens) -> ControlTokenValue {
+        switch token {
+        case .avatarSize:
```

Comment:
activityindicator it is ".side" but in avatar it is ".avatarSize"?

Reply:
Yep! Not sure why other than weird legacy/history.